build_tools/packaging/linux/package.json

```diff
@@ -114,29 +155,33 @@
       "doc"
     ],
     "Artifact": "base",
+    "Artifact_Subdir": "rocprofiler-register",
     "Gfxarch": "False"
 
   },
   {
-    "Package": "amd-llvm",
+    "Package": "rocm-llvm",
     "Version": "",
     "Architecture": "amd64",
     "BuildArch": "x86_64",
     "DEBDepends": [
+      "rocm-core",
       "python3",
       "libc6",
       "libstdc++6|libstdc++8",
       "libstdc++-5-dev|libstdc++-7-dev|libstdc++-11-dev",
       "libgcc-5-dev|libgcc-7-dev|libgcc-11-dev"
     ],
     "RPMRequires": [
+      "rocm-core",
       "python3",
       "glibc",
       "libstdc++",
       "libgcc"
     ],
     "Maintainer": "ROCm Compiler Support <rocm.compiler.support@amd.com>",
-    "Description": "ROCm llvm compiler based on LLVM 20.0.0",
+    "Description_Short": "ROCm llvm compiler",
+    "Description_Long": "ROCm llvm compiler based on LLVM 20.0.0",
```

Comment:
I'm anticipating bitrot in version strings like this, unless we can generate / source the contents from code somehow.